services/QuillLMS/spec/services/clever_integration/classrooms/updater_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe CleverIntegration::Classrooms::Updater do
+  let(:clever_id) { '123_456' }
+
+  let!(:classroom) do
+    create(:classroom,
+      clever_id: clever_id,
+      name: name,
+      synced_name: synced_name
+    )
+  end
+
+  let(:data) do
+    {
+      clever_id: clever_id,
+      name: data_name,
+      grade: '100'
```

Comment:
I'm amused by the idea that `grade` doesn't have some sort of validation to make this impossible to use.

Reply:
I was surprised too given https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/models/classroom.rb#L28.